nss-day-cohort-60/national-parks-client
# Client can now display a list of national parks

# Description
Created a Parks directory that contains a ParksList and Parks component.  The ParksList passes a park prop to the Parks component which then renders the park's name and photo on the Landing page. 

# Steps to test

1. Fetch the vs-parks-list-photo branch and switch to it
2. Start/restart debugger
3. Start the client in the browser
4. Navigate to the landing page and verify that the national parks list is displayed with a photo for each park
